common/scripts/base.sh

```diff
@@ -139,3 +143,27 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
+{
+  if ! bash ${mesos_artifacts_home}/common/${1}/deploy.sh; then
+    echoError "Non-zero exit code returned when deploying ${1}"
+    exit 1
+  fi
+}
+
+function deploy_service()
+{
+  if ! deploy ${1} $self_path/${1}.json; then
+    echoError "Non-zero exit code returned when deploying ${1}"
```

Comment:
Shall we put a msg like "Aborting  deployment" here since specific failure reason is already printed in deploy function? That will give the idea that deployment did not continue after this point